### DISCUSSION TITLE
Media element seek breaking changes with latest version

### DISCUSSION THREAD

**Discussion by ne0rrmatrix** [CommunityToolkit/Maui#1303] [Q&A]
With the change in media element on latest version, I have found seeking just does not work with media element in windows now. I was curious if I have to change my methods with the changes. I noticed it was version bump so I imagine the changes require different user behavior. I saw that in patch notes. But the patch notes did not mention what you have to do differently now? Or if it did I did not see it.

Any suggestions? Oh btw I have not had a chance to test this so my comment about it not working is neither confirmed or tested. Just my own edge case use at the moment does not work. I am using custom controls, inside a content page that are referenced by multiple pages so it could be any number of things. 

Reverting back to previous version fixes the seeking problem but I would prefer to be on latest. If anyone can confirm I am using media element wrong now on the new version or can explain how to seek with the changes I would very much appreciate it.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
